network/manager.go

```diff
@@ -112,6 +114,10 @@ type NetworkManager interface {
 	GetNumberOfEndpoints(ifName string, networkID string) int
 	GetEndpointID(containerID, ifName string) string
 	IsStatelessCNIMode() bool
+	SaveState(eps []*endpoint) error
+	DeleteState(epInfos []*EndpointInfo) error
+	GetEndpointInfosFromContainerID(containerID string) []*EndpointInfo
+	GetEndpointState(networkID, containerID string) ([]*EndpointInfo, error)
```

Comment:
why do we have 2 GET apis. where this is required?

Reply:
use one if in stateless cni and other in stateful cni (during deletion)